streisand

```diff
@@ -52,6 +52,24 @@ function check_python() {
 function check_ssh_key() {
   local STREISAND_SSH_KEY="$HOME/.ssh/id_rsa.pub"
 
+  # In case we're sharing SSH keys inside Docker
```

Comment:
Unfortunately your diffs to `streisand` to support this SSH key stuff is going to have to be reworked on top of https://github.com/StreisandEffect/streisand/pull/970. Apologies.

Reply:
It was based on my broken ideas of how we were handling SSH keys. My current plan is to make a separate wrapper around `./streisand` to handle copying host `$HOME/.ssh` into `/root/.ssh`.

Reply:
Seeing it now I think it would be better to try and only mount the single key required into the container vs the entire `.ssh` directory (https://github.com/StreisandEffect/streisand/pull/980#discussion_r145297893)